examples/seismic/tti/tti_example.py

```diff
@@ -37,13 +37,6 @@ def run(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
     # Define receiver geometry (spread across `x, y`` just below surface)
     rec, u, v, summary = solver.forward(save=save, autotune=autotune)
 
-    if preset == 'constant':
```

Comment:
sorry I'm confused now... why is this being entirely dropped?

Reply:
As I see it, those lines are not longer necessary